tests/test_game_state.py

```
@@ -0,0 +1,367 @@
+    else:
+        assert game_state.active_piece.rotation != original_rotation
+
+
+def test_rotate_piece_invalid(game_state):
+    """Test invalid piece rotation."""
+    # Move piece to edge where rotation would be invalid
+    game_state.active_piece.x = settings.dimensions.BOARD_WIDTH - 1
+    game_state.active_piece.y = settings.dimensions.BOARD_HEIGHT - 1
+
+    # Try to rotate (might be invalid due to collision)
+    success = game_state.rotate_piece(clockwise=True)
```

Ruff (0.14.2)

Local variable `success` is assigned to but never used

Remove assignment to unused variable `success`

(F841)